README.md

```diff
@@ -1,220 +1,60 @@
-    "date_updated": "2023-02-06T02:39:15",
-    "name": "test"
-  },
-  "scope": "com.ibm.aot.eng"
-}
+export FLASK_APP=catalog
+poetry run flask run
 ```
+The above commands will work for zsh and Linux. If you are using another shell or windows, export the variable appropriately. 
 
-_Response body_
+During development, if you want to debug set the debug flag.
 
 ```console
-{
-  "_id": "63ed5b84ca55ee384494b3a7",
-  "data_type": "1",
-  "data_uri": "data_uri",
-  "metadata": {
-    "date_created": "2023-02-06T02:39:15",
-    "date_updated": "2023-02-06T02:39:15",
-    "name": "test"
-  },
-  "scope": "com.ibm.aot.eng"
-}
+export FLASK_DEBUG=1
 ```
+The difference between the two is that exceptions are exposed in detail in development mode.
 
-* **Update a specific asset in the catalog**
-
-```console
-$ curl -X 'PUT' \
-  'http://127.0.0.1:5000/sda/update/63ed5b84ca55ee384494b3a7' \
-  -H 'accept: application/json' \
-  -H 'Content-Type: application/json' \
-  -d '{
-  "data_type": "3",
-  "data_uri": "data_uri_update"
-}'
-```
+Once the server is running, you can access the REST API for self-describing entries. 
+In development mode, the default server is at http://127.0.0.1:5000. The REST API is accessible via the prefix http://127.0.0.1:5000/sda. 
+The user management console is located at http://127.0.0.1:5000/user/index.
 
-_Response body_
+## Running the Application with Docker
 
-```console
-{
-  "_id": "63ed5b84ca55ee384494b3a7",
-  "data_type": "3",
-  "data_uri": "data_uri_update",
-  "metadata": {
-    "date_created": "2023-02-06T02:39:15",
-    "date_updated": "2023-02-06T02:39:15",
-    "name": "test"
-  },
-  "scope": "com.ibm.aot.eng"
-}
-```
+The application can also be run using containers. You need to have the ability to build images locally and run them as containers. The current version has been tested using Docker Desktop.
 
-* **Delete a specific asset from the catalog**
+To bring up the application using Docker Compose:
 
 ```console
-$ curl -X 'DELETE' \
-  'http://127.0.0.1:5000/sda/delete/63ed5b84ca55ee384494b3a7' \
-  -H 'accept: */*'
-```
-
-_Response body_
+$ cd sd-catalog-backend
 
-```console
-"Asset deleted successfully!"
+$ docker-compose up -d --build
 ```
+This will bring up two containers 
++ sd-catalog-backend - containing the application logic and an embedded sqlite datastore for user management
++ mongodb - managing the self describing catalogue entries
 
-<hr>
-
-To update the application during development you need to use the following commands to bring the app down and remove the
-backend volume:
-```console
-$ docker-compose down
-Removing sd-catalog-backend ... done
-Removing mongodb            ... done
-Removing network sd-catalog-backend_backend
-Removing network sd-catalog-backend_frontend
-
-$ docker volume ls
+The swagger documentation is avalable at http://127.0.0.1:5000/apispec/
 
-$ docker volume rm sd-catalog-backend_appdata
-```
\ No newline at end of file
+Below are the available APIs. 
```

Comment:
Looks like you removed all the curl commands outlining the APIs and output. Can you please add them back. Also can you please add back the debugging instructions that is at the end of the current README

Reply:
Thx for pointing that out. Corrected.